nengo_loihi/loihi_cx.py

```diff
@@ -101,17 +102,25 @@ def add_probe(self, probe):
         assert probe.target is self
         self.probes.append(probe)
 
-    def set_decay_U(self, tau_s, dt):
-        self.decayU[:] = 1 if tau_s == 0 else -np.expm1(-dt/np.asarray(tau_s))
+    def set_decay_U(self, tau_s, dt, default=False, overwrite=False):
```

Comment:
Is `overwrite` something that we already use somewhere? Looking through the diff I don't see it being used, and because of that I would opt to not include this kwarg as I feel like it makes the function a bit harder to read. Also, it feels like if we end up needing to use it, we should instead refactor a bit so that we don't need to use.

Reply:
Those are fair points. I had included it just because I generally don't like methods that don't let me do what I want even if I'm saying "this is what I want". I sometimes find that causes me to hack things by just looking at the internals to the method and calling them manually. But I do agree that we should hopefully be able to not overwrite, and if we do run into a sticky situation we can always figure it out later. I'll take it out.

Reply:
I agree in principle that you don't want to look at places other than the method to do what you want, but in this case it's only being used internally anyhow so we can expect some familiarity with the class. In any case, we can revisit later if we run into situation where we really want to overwrite.